java-logging-appinsights/build.gradle

```diff
@@ -3,8 +3,9 @@ plugins {
 }
 
 def versions = [
-  springBoot: '2.0.8.RELEASE',
-  appInsights: '2.1.2'
+  springBoot: '2.1.2.RELEASE',
+  appInsights: '2.3.1',
```

Comment:
Is this definition used after moving to starter?

Reply:
yes, we still have dependancy to add applicationinsights-logging-logback as the starter doesn't configure .